autoPlay.js

```diff
@@ -273,6 +273,7 @@ function goToLaneWithBestTarget() {
 				// ignore if lane is empty
 				if(g_Minigame.CurrentScene().m_rgGameData.lanes[i].dps == 0)
 					continue;
+					rainingGold = false;
```

Comment:
Apologies, i didn't scroll horizontally to see the if statement had no opening brace and this line is indented with the line above so as to appear in a block. Ignore me. :)